### PR TITLE
Update to 4.9

### PR DESCRIPTION
**Pull request by telphan**
Connects to resin-os/resinos#296

### CODE REVIEW

File: layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard_4.9.bb [hunk 1]
```diff
@@ -21,9 +21,9 @@ COMPATIBLE_MACHINE = "beaglebone"
 
 S = "${WORKDIR}/git"
 
-TAG = "4.4.55-ti-rt-r94"
+TAG = "4.9.27-ti-rt-r33"
```
Comment (agherzan) [inline layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard_4.9.bb]: We decided to switch off rt.

**Comment by floion** [comment]
@telphan I see that the tag 4.9.27-ti-r33 is missing from the kernel repo now. I have opened an issue here https://github.com/beagleboard/linux/issues/126 to keep track of why this has happened.
In the meanwhile, please use the tag 4.9.30-ti-r38 in this PR.
Also, the current PR uses the older defconfig for the new kernel whish should not happen. Make sure you also copy the appropriate defconfig like we did here: https://github.com/resin-os/resin-beaglebone/commit/25b99de330c1cb4b99419d310d3e0a65341980db

**Comment by telphan** [comment]
@floion not sure why the tag does not appear in github, but the repo has that tag.

File: layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard-4.9/defconfig [hunk 2]
```diff
@@ -2,8 +2,6 @@
 CONFIG_KERNEL_LZO=y
```
Comment (floion) [inline layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard-4.9/defconfig]: Commit log mentions 4.4 when it should mention 4.9

**Pull request merged**